src/status_im/contexts/settings/wallet/keypairs_and_accounts/actions/view.cljs

```diff
@@ -1,44 +1,53 @@
 (ns status-im.contexts.settings.wallet.keypairs-and-accounts.actions.view
   (:require [quo.core :as quo]
+            [react-native.core :as rn]
             [status-im.contexts.settings.wallet.keypairs-and-accounts.remove.view :as remove-key-pair]
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
-(defn on-rename-key-pair
-  [key-pair]
-  (rf/dispatch [:open-modal :screen/settings.rename-keypair key-pair]))
-
-(defn on-remove-key-pair
-  [key-pair]
-  (rf/dispatch [:show-bottom-sheet
-                {:theme   :dark
-                 :content (fn []
-                            [remove-key-pair/view key-pair])}]))
-
-(defn on-show-qr
-  [data]
-  (rf/dispatch [:open-modal :screen/settings.encrypted-key-pair-qr data]))
-
 (defn view
-  [props key-pair]
-  (let [has-paired-device (rf/sub [:pairing/has-paired-devices])]
+  [props keypair]
+  (let [has-paired-device (rf/sub [:pairing/has-paired-devices])
+        missing-keypair?  (rn/use-memo #(= (:stored props) :missing) [props])
```

Comment:
We might not need to memo here? Or at least if we must memo, we probably only want to memo based on the `:stored` prop. Wdyt?

Reply:
yeah actually it doesn't do any calculation and i can remove the memo